src/docfx/lib/js/chakra/JavaScriptRuntimeAttributes.cs

```diff
@@ -40,19 +40,21 @@ public enum JavaScriptRuntimeAttributes
         /// <summary>
         ///     Using Eval or Function constructor will throw an exception.
         /// </summary>
-        [SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1650:ElementDocumentationMustBeSpelledCorrectly", Justification = "Eval is a valid function name.")]
+        [SuppressMessage(
+            "StyleCop.CSharp.DocumentationRules",
+            "SA1650:ElementDocumentationMustBeSpelledCorrectly",
```

Comment:
I didn't find any SA1650 message after removed this, please help confirm if we still keep this.

Reply:
oh, don't touch the files under chakra folder, it is copied from elsewhere, there is a README.md explaining that

Reply:
There is a GlobalSuppressions.cs file that suppresses warnings in chakra folder

Reply:
Got it, I will add MEN002 to there.

Reply:
Reverted the changes for files under chakra folder